sample-btcd.conf

```diff
@@ -293,8 +293,7 @@
 ; worth your while.
 ; generate=false
 
-; Add addresses to pay mined blocks to for CPU mining and the block templates
-; generated for the getwork RPC as desired.  One address per line.
+; Add addresses to pay mined blocks to for CPU mining.  One address per line.
```

Comment:
I should note that it is **potentially** true though.  It depends on the flags in the getblocktemplate request.  It is possible to request templates with or without coinbase details.  When requested with coinbase details, a mining address is required.